st2common/st2common/models/db/rbac.py

```diff
@@ -76,9 +82,26 @@ class PermissionGrantDB(stormbase.StormFoundationDB):
     permission_types = me.ListField(field=me.StringField())
 
 
+class GroupToRoleMappingDB(stormbase.StormFoundationDB):
```

Comment:
I can add it if needed, but those DB models itself don't implement any custom functionality or methods so I think such tests are very low value.

We already have tests for other important stuff and with testing model create / save / update we are basically just testing mongoengine functionality and we already test that in other places.